R/mod_ParticipantMetricSummary_UI.R

```diff
@@ -8,7 +8,11 @@ mod_ParticipantMetricSummary_UI <- function(id) {
   ns <- NS(id)
   out_Card(
     "Metric Summary",
-    uiOutput(ns("metric_list")),
+    shinycssloaders::withSpinner(
+      uiOutput(ns("metric_list")),
+      type = 7,
+      id = ns("spinner")
```

Comment:
provide a more precise name, like `metric_list-spinner`? in the event that we add even more spinners